### PR TITLE
Added types for js-money

### PR DESCRIPTION
**Pull request by kanatkubash**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not provide its own types, and you can not add them.
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/js-money/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    static XAF: Currency;
+    static XOF: Currency;
+    static YER: Currency;
+    static ZAR: Currency;
+    static ZMK: Currency;
+    //#endregion
+}
+
+/// to supress new Money() error
+declare namespace Money { }
```
Comment (DanielRosenwasser) [inline types/js-money/index.d.ts]: This is an inappropriate hack. [Please read more here from the DefinitelyTyped REAMDE](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/fbba9f55beab742b9d1c0fa9ef714e0771ad9d05#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports)

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/js-money/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    | 'JPY' | 'KES' | 'KHR' | 'KMF' | 'KRW' | 'KWD' | 'KZT'
+    | 'LAK' | 'LBP' | 'LKR' | 'LTL' | 'LVL' | 'LYD' | 'MAD'
+    | 'MDL' | 'MGA' | 'MKD' | 'MMK' | 'MOP' | 'MUR' | 'MXN'
+    | 'MYR' | 'MZN' | 'NAD' | 'NGN' | 'NIO' | 'NOK' | 'NPR'
+    | 'NZD' | 'OMR' | 'PAB' | 'PEN' | 'PHP' | 'PKR' | 'PLN'
+    | 'PYG' | 'QAR' | 'RON' | 'RSD' | 'RUB' | 'RWF' | 'SAR'
+    | 'SDG' | 'SEK' | 'SGD' | 'SOS' | 'SYP' | 'THB' | 'TND'
+    | 'TOP' | 'TRY' | 'TTD' | 'TWD' | 'TZS' | 'UAH' | 'UGX'
+    | 'UYU' | 'UZS' | 'VEF' | 'VND' | 'XAF' | 'XOF' | 'YER'
+    | 'ZAR' | 'ZMK' | string;
```
Comment (DanielRosenwasser) [inline types/js-money/index.d.ts]: This union is now effectively just `string` (CC @RyanCavanaugh).

File: types/js-money/index.d.ts (hunk 1 above)
Comment (kanatkubash) [reply types/js-money/index.d.ts]: Thanks for pointing out the errors. Didn't know of such construction
I'll update accordingly

File: types/js-money/index.d.ts (hunk 2 above)
Comment (kanatkubash) [reply types/js-money/index.d.ts]: Do you know is there a way to accept both string literals and strings ? 
Thanks

**Comment by kanatkubash** [comment]
@DanielRosenwasser requesting review

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
Thank you!

**Pull request merged**